packages/sync/src/Queue.php

```diff
@@ -506,7 +693,7 @@ private function validate_checkout( $buffer ) {
 			return new \WP_Error( 'buffer_not_checked_out', 'There are no checked out buffers' );
 		}
 
-		if ( $checkout_id != $buffer->id ) {
+		if ( $checkout_id !== $buffer->id ) {
```

Comment:
are we sure both are the same type?

Reply:
They are both supposed to be strings. So yes.

Reply:
nope... `$buffer->id` is an `int` ---

Reply:
let's revert this change just in case we broke something?